CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- n/a
+### Fixed
+- Fixed minor bug in map_404_to_none, where the response could have response equals to None
```

Comment:
I don't think this kind of changelog entry makes sense, because there is no map_404_to_none in the API.  Can you please describe this in terms that an end-user of the library could understand?  How did this bug affect any code using the library?

Reply:
And since https://github.com/release-engineering/pubtools-pulplib/pull/51 was fixing the same bug in a different piece of code, I guess the commit message / changelog entry there includes some hints.